modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/access_from_desktop.jsp

```diff
@@ -30,6 +30,11 @@ DLAccessFromDesktopDisplayContext dlAccessFromDesktopDisplayContext = new DLAcce
 	<div class="portlet-document-library">
 		<liferay-ui:message key="<%= dlAccessFromDesktopDisplayContext.getWebDAVHelpMessage() %>" />
 
+		<liferay-learn:message
```

Comment:
@ericyanLr 
I did not know about this component.
Interesting!

Thanks for the info :)